include/swift/AST/DiagnosticsSema.def

```diff
@@ -2533,8 +2533,11 @@ NOTE(silence_inject_forced_downcast,none,
      "add parentheses around the cast to silence this warning", ())
 
 ERROR(conditional_downcast_foreign,none,
-      "conditional downcast to CoreFoundation type %0 will always succeed",
+     "conditional downcast to CoreFoundation type %0 will always succeed",
       (Type))
+NOTE(compare_CFTyteId_foreign,none,
+     "compare CFTypeID between %0 and %1 to make sure they are convertible",
```

Comment:
How about

> did you mean to explicitly compare the CFTypeIDs of %0 and %1?

Reply:
Even this isnt necessarily a good wording for this.  The type of the cast variable is statically known to the type checker, but the type ID of the cast variable isn’t known until runtime.  %0 should mention the variable, not necessarily the type we inferred here.

Reply:
Thanks! I am trying to get variable name for %0 here but not sure how to do it. Does it need to be an `Identifier` type? And how can I get it from `cast` or `sub` inside the method `getReferencedDecl`?